manifests/profile/puppet/master.pp

```diff
@@ -99,7 +100,7 @@
     }
   }
 
-  tidy { '/opt/puppetlabs/server/data/puppetserver/reports':
+  tidy { $reports_dir:
```

Comment:
I'd like a double-check that this has been extracted. Seems a little odd to see it on this PR.

Reply:
Errant commit on this PR; removed and force-pushed.